migrate2rocky/migrate2rocky.sh

```diff
@@ -283,6 +283,7 @@ pre_check () {
 
 	dir=${dirs[$((i++))]}
 
+        # shellcheck disable=SC2034
```

Comment:
I don't know why this variable is getting defined and set a value, but has not been used in script, maybe something for later?
Let's ignore shellcheck warning for now.

Reply:
Go ahead and remove the variable here, it's not likely going to be needed, and if it is we can always add it back later.

Reply:
Removed in https://github.com/rocky-linux/rocky-tools/pull/146/commits/6449dfa18da2c6cb3502660751ad616f28c949a6.